scripts/BHM-example-mpi.jl

```diff
@@ -1,5 +1,11 @@
-# In this example, we will demonstrate using Rimu with MPI.  A runnable script is located at
-# [here](../../scripts/BHM-example-mpi.jl). Run it as `mpirun julia BHM-example.jl`.
+# # Example: Rimu with MPI
+
+# In this example, we will demonstrate using Rimu with MPI. 
+
+# A runnable script for this example is located [here](../../../scripts/BHM-example-mpi.jl). 
```

Comment:
Same problem with the link to the script as for BHM-example.jl, please see the other comment down below.